common/src/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import { Meta } from "@storybook/react";
+import React from "react";
+import Button from "./Button";
+
+export default {
+  title: "Components/Button",
+  component: Button,
+} as Meta;
+
+export const PrimaryButton = () => (
```

Comment:
Is there another issue for creating all of the different button variations in storybook? These variations used to exist in storybook.

Reply:
I dunno what happened to them :/

Reply:
That's my fault, I forgot to replace it.